x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -301,14 +301,28 @@ private static Map<String, RoleDescriptor> initializeReservedRoles() {
                     "Grants access to manage all index templates and all ingest pipeline configurations."
                 )
             ),
-            // reporting_user doesn't have any privileges in Elasticsearch, and Kibana authorizes privileges based on this role
             entry(
                 "reporting_user",
                 new RoleDescriptor(
                     "reporting_user",
                     null,
                     null,
-                    null,
+                    new RoleDescriptor.ApplicationResourcePrivileges[] {
+                        RoleDescriptor.ApplicationResourcePrivileges.builder()
+                            .application("kibana-.kibana")
+                            .resources("*")
+                            .privileges(
+                                "feature_discover.minimal_read",
```

Comment:
Is this list just copied from the jira? That's the only place I could find it :) 

Also, looks like
```
    "transient_metadata": {
      "enabled": true
    }
```

Is set in the jira. Should that be set here too?

Reply:
NVM, it's set to that by default 
```
        this.transientMetadata = transientMetadata != null
            ? Collections.unmodifiableMap(transientMetadata)
            : Collections.singletonMap("enabled", true);
```